### PR TITLE
upgrade Keystone to mitaka version (and bumps to 1.5.0) 

### PR DESCRIPTION
**Pull request by AlvaroVega**
Do not merge until:
- [x] Liberty was merged into master: https://github.com/telefonicaid/fiware-keystone-spassword/pull/7
- [x] Docker should be compatible with mitaka
- [x] keystone-scim should be compatible with mitaka:
     https://github.com/telefonicaid/fiware-keystone-scim/issues/43
- [x] keystone-spassword packages were compatible with mitaka:
     https://github.com/telefonicaid/fiware-keystone-spassword/issues/38
- [x] LDAP Tested:
- [x] 2FA Tested:

'extra' field has dissapear from mitaka version:
https://bugs.launchpad.net/keystone/+bug/1576765

According with mitaka keystone version release notes:
http://docs.openstack.org/releasenotes/keystone/mitaka.html
[blueprint removed-as-of-mitaka] Removed extras from token responses. These fields should not be necessary and a well-defined API makes this field redundant. This was deprecated in the Kilo release.

This PR (Mitaka version upgrade) implies also a new version of SCIM plugin https://github.com/telefonicaid/fiware-keystone-scim/pull/44

### CODE REVIEW

**Comment by AlvaroVega** [comment]
To work again with extras ensure token/providers/common.py
```
  def get_token_data(self, user_id, method_names, extras=None, domain_id=None,
                       project_id=None, expires=None, trust=None, token=None,
                       include_catalog=True, bind=None, access_token=None,
                       issued_at=None, audit_info=None):
        if extras is None:
            extras = {}
        token_data = {'methods': method_names,
                      'extras': extras}
```

File: keystone_spassword/contrib/spassword/backends/sql.py [hunk 1]
```diff
@@ -27,8 +27,8 @@
 from keystone_spassword.contrib.spassword import Driver
```
Comment (fgalan) [inline keystone_spassword/contrib/spassword/backends/sql.py]: Does this PR needs of changelog entry?

**Comment by AlvaroVega** [comment]
Since Mitaka there is no way to get domain_id if previous data exists:
https://github.com/telefonicaid/fiware-keystone-spassword/blob/master/docker/postlaunchconfig_update.sh#L44

File: keystone_spassword/contrib/spassword/backends/sql.py (hunk 1 above)
Comment (AlvaroVega) [reply keystone_spassword/contrib/spassword/backends/sql.py]: fixed in 3eb8d9f9dd9eef8bf8f5dec0ba828c389b80ac7b

**Comment by AlvaroVega** [comment]
@fgalan This PR is ready to be merged.

File: docker/Dockerfile_mitaka [hunk 2]
```diff
@@ -0,0 +1,105 @@
+FROM centos:centos7
```
Comment (fgalan) [inline docker/Dockerfile_mitaka]: Temp file. To be deleted.

Comment (AlvaroVega) [reply docker/Dockerfile_mitaka]: fixed in 6b7b3d5a8aa2dfa47964b9ebf0fa5686b0d98ea0

**Comment by fgalan** [comment]
LGTM

**Pull request merged**